### ISSUE TITLE
Checking account classification failes since 23.07.2020

### ISSUE THREAD

**Issue by grindsa** [grindsa/dkb-robo#19]
Checking account classification failes as DKB removed the "Cash im Shop" link  in account overview. As result `get_transactions()` failes with the below error

```
    TRANSACTION_DIC[1] = dkb.get_transactions(dkb.account_dic[0]['transactions'], dkb.account_dic[0]['type'], YESTERDAY, TODAY)
  File "/usr/lib/python2.7/site-packages/dkb_robo/dkb_robo.py", line 430, in get_transactions
    transaction_list = self.get_creditcard_transactions(transaction_url, date_from, date_to)
  File "/usr/lib/python2.7/site-packages/dkb_robo/dkb_robo.py", line 107, in get_creditcard_transactions
    self.dkb_br.select_form('#form1579108072_1')
  File "/usr/lib/python2.7/site-packages/mechanicalsoup/stateful_browser.py", line 207, in select_form
    raise LinkNotFoundError()
mechanicalsoup.utils.LinkNotFoundError
```

**Comment by grindsa** [comment]
Quick fix daf35c9 has been pushed to devel branch

**Comment by grindsa** [comment]
fxed in [0.10.3](../releases/0.10.3)
